src/import-data/useImportRequest.test.ts

```diff
@@ -95,20 +106,84 @@ describe('getImportRequest', () => {
     {
       queryParams: {
         format: 'snapshot',
-        snapshotIds: ['00001111-2222-3333-aaaa-bbbbccccdddd', 'aaaabbbb-cccc-1111-2222-333333333333'],
+        snapshotIds: ['aaaabbbb-cccc-1111-2222-333333333333', '00001111-2222-3333-aaaa-bbbbccccdddd'],
```

Comment:
Just curious!  Why did you swap these values?

Reply:
No real reason. When I added the catalog request to this function, I think I mistakenly changed these to the dataset IDs instead of their `dct:identifier`s. And when I changed them back, I put them in the reverse order without realizing it.

Reply:
If it was easy to assert the result in an order independent way, that wouldn't be a bad thing to do, since I don't think we need to require the code to always preserve order.

Reply:
You're right, order isn't important here and I ran into that and was annoyed by it while writing this test.

> If it was easy to assert the result in an order independent way
After this comment, I explored using a custom expect "matcher" to do this. I came up with what I think is a useful test helper, but it's enough code that I think it warrants its own PR.